src/js/common/components/form-elements/GrowableTable.jsx

```diff
@@ -106,26 +119,30 @@ class GrowableTable extends React.Component {
                   })}
               </div>
               <div className="small-3 columns">
-                <button className="usa-button-outline short" onClick={(event) => this.handleEdit(event)} data-key={obj.key}><i className="fa before-text fa-pencil"></i>Edit</button>
```

Comment:
Yes, @goldenmeanie thought they weren't adding anything and were taking up space.